contracts/stabilizer/IPegStabilityModule.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../pcv/IPCVDeposit.sol";
+
+/**
+ @title Fei Peg Stability Module
+ @author Fei Protocol
+
+ The Fei PSM is a contract which holds a reserve of assets in order to exchange FEI at $1 of underlying assets with a fee.
+ * `mint()` - buy FEI for $1 of underlying tokens
+ * `redeem()` - sell FEI back for $1 of the same
+
+ The contract has a reservesThreshold() of underlying meant to stand ready for redemptions. Any surplus reserves can be sent into the PCV using `allocateSurplus()`
+
+ The contract is a 
+ * PCVDeposit - to track reserves
+ * OracleRef - to determine price of underlying, and 
+ * RateLimitedMinter - to stop infinite mints and related issues.
+
+ Idea inspired by MakerDAO PSM, code written without reference
+ */
+interface IPegStabilityModule {
+    
+    // ----------- State changing Api -----------
+
+    /// @notice mint `amountFeiOut` FEI to address `to` for `amountIn` underlying tokens
+    /// @dev see getMintAmountOut() to pre-calculate amount out
+    function mint(address to, uint256 amountIn)
+        external
+        returns (uint256 amountFeiOut);
+
+    /// @notice redeem `amountFeiIn` FEI for `amountOut` underlying tokens and send to address `to` 
+    /// @dev see getRedeemAmountOut() to pre-calculate amount out
+    function redeem(address to, uint256 amountFeiIn)
```

Comment:
Same comment here. There should be a third param that allows users to specify their output currency so that they can receive assets of their choosing.  @Joeysantoro Please confirm that a single PSM contract will be responsible for handling all assets.